lms/static/scripts/frontend_apps/components/test/LMSGrader-test.js

```diff
@@ -81,6 +82,52 @@ describe('LMSGrader', () => {
     );
   });
 
+  it('orders the students by last name (if it exists), then first name', () => {
+    // Un-order students
+    fakeStudents = [
+      {
+        displayName: 'Student Beta',
+      },
+      {
+        displayName: 'Students Beta',
+      },
+      {
+        displayName: 'Beta',
+      },
+      {
+        displayName: 'student Beta',
+      },
+      {
+        displayName: 'Student Delta',
+      },
+      {
+        displayName: 'Student Alpha',
+      },
+      {
+        displayName: 'Alpha',
+      },
+    ];
+    const wrapper = renderGrader();
+    const orderedStudentNames = wrapper
+      .find('FakeStudentSelector')
+      .prop('students')
+      .map(s => {
+        return s.displayName;
+      });
+    assert.match(
+      [
+        'Alpha',
+        'Beta',
+        'Student Alpha',
+        'Student Beta',
+        'student Beta',
```

Comment:
Is it simply because it comes later in the unsorted `Array` that 'student Beta' ends up after 'Student Beta'?

Reply:
correct, these are a tie, or its initial order actually matters. I'll add a comment